src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java

```diff
@@ -24,12 +24,14 @@
      * Type B is a class that extends {@link AbstractBuildingWorker.View}.
      */
     protected final B          building;
-    private final   SwitchView switchView;
     private final   Label      title;
-    private final   Button     buttonPrevPage;
-    private final   Button     buttonNextPage;
     private final   Button     buttonBuild;
     private final   Button     buttonRepair;
+    protected       SwitchView switchView;
+    protected       int        switchPagesSize;
```

Comment:
### Errors:
- Protected field switchPagesSize 
- Can be package-private

### Warnings:
- Instance field name switchPagesSize doesn't match regex 'm_[a-z][A-Za-z\d]*'